product_specifications/12_xfomv_e/readme.md

```diff
@@ -0,0 +1,46 @@
+
+## Problem
+
+Sometimes members or consortium organizations choose to discontinue their use of DataCite DOI services, this results in inactive repositories and management of the DOIs. We currently have 56 inactive repositories, which includes 449,514 DOIs in Fabrica that are no longer managed by members. This was largely due to the transition in the membership model and we continue to work with these organizations to find solutions. 
+
+
+Problem Validation: 
+
+closes: https://github.com/datacite/tasks/issues/46
+
+## Proposed Solution 
+
+Use an inactive workflow with a series of features to deal with the departing members.
+
+- [Wire Frame]()
+- Solution Validation: 
+
+## User Stories
+
+### CRITICAL
+
+- [ ] As a Datacite staff, I would like to be able set a repository inactive directly from fabrica, so I don't have to worry about any other consequences of the transition to inactivity.
+- [ ] As a Datacite staff, I would like that DOIs from a inactive repository were resolving to their urls as long as the urls are the correct, so that we can keep the P in PIDs.
```

Comment:
Maybe we would need to trigger some R&D activities to find a way or identify a compromise on how to check the links. 

We were using link checking to manage that link rot decay but unfortunately link checking is not very effective. The average number of checks that can be done are 1000s (@richardhallett has comment on that regard in the other doc  ) on a day and if done to the same server it can lead of a denial of service (the server blocking the link checker).

Reply:
Yes, I'm suggesting improving the link checker independent to this work.